back/engines/commercial/report_builder/app/models/report_builder/report.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: report_builder_reports
+#
+#  id         :uuid             not null, primary key
+#  name       :string           not null
+#  owner_id   :uuid             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_report_builder_reports_on_name      (name) UNIQUE
+#  index_report_builder_reports_on_owner_id  (owner_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (owner_id => users.id)
+#
+module ReportBuilder
+  class Report < ::ApplicationRecord
+    belongs_to :owner, class_name: 'User'
+    has_one(
```

Comment:
Being a bit thick here. Where does the relationship between reports and layouts get persisted?

Reply:
`has_one` means it is persisted on the other side, in the other table. In this case, it is stored in the `content_buildable_id`and `content_buildable_type` columns of the `content_builder_layouts` table.